src/plugins/input_control_vis/public/control/list_control_factory.ts

```diff
@@ -190,7 +190,7 @@ export class ListControl extends Control<PhraseFilterManager> {
       return;
     }
 
-    this.partialResults = resp.terminated_early || resp.timed_out;
+    this.partialResults = resp.timed_out;
```

Comment:
Why was this changed? I think there are two valid use cases here: When the request hits either the `timeout` (time limit) specified or the `terminate_after` (# of docs limit) value that is sent.

Reply:
My bad, it's another argument missing from the es client type definition.

Restoring and opening an issue.